README.md

```diff
@@ -359,6 +359,14 @@ breakpoints: ?Object = {lg: 1200, md: 996, sm: 768, xs: 480, xxs: 0},
 // # of cols. This is a breakpoint -> cols map, e.g. {lg: 12, md: 10, ...}
 cols: ?Object = {lg: 12, md: 10, sm: 6, xs: 4, xxs: 2},
 
+// # of margin. This is a breakpoint -> margin map, e.g. {lg: [10, 10], md: [10, 10], ...}
+margin: ?Object = {lg: [10, 10], md: [10, 10], sm: [10, 10], xs: [10, 10], xxs: [10, 10]},
```

Comment:
we definitely should keep backward compatibility so we should change that type (in my opinion) like this:

```
[number, number] | ?Object
```

and we should do it for containerPadding as well